### DISCUSSION TITLE
HTML super languages support

### DISCUSSION THREAD

**Discussion by lucaspmarra** [biomejs/biome#4314] [General]
One year after launch!
And it wasn't until version 1.6.0 that we had [HTML super languages support](https://biomejs.dev/internals/language-support/#html-super-languages-support) 

Are there any plans to have full support without the need for small flow changes (workarounds)?

**Comment by dyc3** [comment]
Yes, we plan to fully support these. Right now, we are aiming to support Vue, Svelte, Astro, and Angular (I plan to do it in that order).

The main blocker right now is embedded language parsing/formatting. You can subscribe to that issue to keep tabs on full support. (I can't link you right now, on mobile)

**Comment by Netail** [comment]
Hey!
As of [Biome 2.3](https://biomejs.dev/blog/biome-v2-3/) we support HTML, Astro, Svelte & Vue under the experimental option: `html.experimentalFullSupportEnabled`.
Support is still experimental due to potentially missed edge cases, but definitely worth trying :) 
In case you find any issues please make sure to [create a GitHub issue](https://github.com/biomejs/biome/issues/new/choose) so we can get closer to stable :)

More on HTML-ish languages can be found here: https://biomejs.dev/internals/language-support/#html-super-languages-support
